test/integration/connect/envoy/case-mesh-to-lambda/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -eEuo pipefail
+
+# Copy lambda config files into the register dir
+# TODO: not sure if we want to make this step more generic and add it to the init_workdir func?
+find ${CASE_DIR} -maxdepth 1 -name 'lambda_*.json' -type f -exec cp -f {} workdir/${CLUSTER}/register \;
```

Comment:
I put this in the setup for the `mesh-to-lambda` case because using JSON and the Consul HTTP API is currently specific to the registration of the lambda functions. The registration of the lambdas requires the use of the Consul HTTP API because the CLI doesn't support registering the lambdas "services" to the catalog.

I didn't think through this very far but we could probably massage this to be more generic if we think it will be useful for other cases in the future.